openwisp_radius/tests/test_selenium.py

```diff
@@ -0,0 +1,252 @@
+import os
+
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from django.urls import reverse
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import Select
+
+from openwisp_users.tests.utils import TestOrganizationMixin
+from openwisp_utils.tests.selenium import SeleniumTestMixin
+
+current_script_path = os.path.dirname(os.path.abspath(__file__))
```

Comment:
I don’t think this is the most effective way to identify the CSV file paths.

Instead, you can use the provided utility and place all the CSV files under `openwisp_radius/tests/static/*.csv`.

https://github.com/openwisp/openwisp-radius/blob/2f6a7cb8e785b8927251418653bbc15f06204ddd/openwisp_radius/tests/test_commands.py#L122